src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -935,11 +872,6 @@ protected final void closeNoLock(String reason) throws ElasticsearchException {
         if (isClosed.compareAndSet(false, true)) {
             assert rwl.isWriteLockedByCurrentThread() || failEngineLock.isHeldByCurrentThread() : "Either the write lock must be held or the engine must be currently be failing itself";
             try {
-                try {
```

Comment:
do you mean fsync it? it will be done when the index shard is closed (the shard owns the translog now..)